aio_openapi.egg-info/PKG-INFO

```diff
@@ -0,0 +1,11 @@
+Metadata-Version: 1.0
```

Comment:
We should have "*.egg-info"in the .gitignore. None of this needs to be in git.

Reply:
Sure. I'll also add it on the other repos